DATA605/TutorialsTask76_automate_collaborator_invitations_from_gsheet.py

```diff
@@ -0,0 +1,82 @@
+"""
```

Comment:
This can be a "master" script since we can reuse it for various applications (next year, fall class, etc).

We should separate the script code from the other function in a `helpers/hgithub.py` library

Reply:
I'll close this to merge. But pls do a follow up.